src/components/Navbar/NavBar.tsx

```diff
@@ -5,6 +5,7 @@ import MenuIcon from '@material-ui/icons/Menu';
 import CloseIcon from '@material-ui/icons/Close';
 import { Logo } from '../Logos';
 import { useTranslation } from 'react-i18next';
+import { AnyAaaaRecord } from 'dns';
```

Comment:
What is this?

Reply:
I'm not sure if I added it. should I delete it ?

Reply:
Maybe an accident import. It's not used anywhere so you can remove it.